indico/htdocs/js/indico/widgets/person_link_widget.js

```diff
@@ -122,44 +198,43 @@
+                        updatePersonOrder();
+                    }
+                    $field.principalfield('set', person.id, targetData);
+                });
+            }
+
             if (options.allow.authors) {
-                if (person.authorType != options.authorTypes.primary) {
-                    var $authorButton = $('<div>').addClass('action-row').text($T.gettext("Move to authors"));
-                    $authorButton.on('click', function() {
-                        $element.qbubble('hide');
-                        $field.principalfield('set', person.id, {authorType: 1});
-                    });
-                    $buttons.append($authorButton);
+                if (person.authorType !== options.authorTypes.primary) {
+                    $buttons.append(actionButton($T.gettext("Move to authors"), $authorList, {authorType: 1}));
                 }
 
-                if (person.authorType != options.authorTypes.secondary) {
-                    var $coAuthorButton = $('<div>').addClass('action-row').text($T.gettext("Move to co-authors"));
-                    $coAuthorButton.on('click', function() {
-                        $element.qbubble('hide');
-                        $field.principalfield('set', person.id, {authorType: 2});
-                    });
-                    $buttons.append($coAuthorButton);
+                if (person.authorType !== options.authorTypes.secondary) {
+                    $buttons.append(actionButton($T.gettext("Move to co-authors"), $coauthorList, {authorType: 2}));
                 }
 
-                if (person.authorType != options.authorTypes.none) {
-                    var $nonAuthorButton = $('<div>').addClass('action-row').text($T.gettext("Move to others"));
-                    $nonAuthorButton.on('click', function() {
-                        $element.qbubble('hide');
-                        $field.principalfield('set', person.id, {authorType: 0, isSpeaker: true});
-                    });
-                    var $speakerButton = $('<div>').addClass('action-row');
-                    $speakerButton.text(person.isSpeaker ? $T.gettext("Not a speaker anymore") : $T.gettext("Make a speaker"));
-                    $speakerButton.on('click', function() {
-                        $element.qbubble('hide');
-                        $field.principalfield('set', person.id, {isSpeaker: !person.isSpeaker});
-                    });
-                    $buttons.append($nonAuthorButton).append($buttonsSeparator).append($speakerButton);
+                if (person.authorType !== options.authorTypes.none) {
+                    $buttons.append(actionButton($T.gettext("Move to others"), $otherList,
+                                                 {authorType: 0, isSpeaker: true}));
+
+                    var text = person.isSpeaker ? $T.gettext("Not a speaker anymore") :
+                                                  $T.gettext("Make a speaker");
```

Comment:
Maybe one of the advanced [operator-linebreak](http://eslint.org/docs/rules/operator-linebreak) settings